src/main/webapp/destinationCreation.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8"/>
+    <title>Destination Creation</title>
+    <link rel="stylesheet" href="destinationCreation.css"/>
```

Comment:
are you missing the css file for this PR?

Reply:
Oops. I forgot to add that file to this PR. I will add that in. Thanks